[SDK-2290] Add support for Organizations (#720)
auth0/auth0-spa-js: README.md

### Organizations (Closed Beta)

Organizations is a set of features that provide better support for developers who build and maintain SaaS and Business-to-Business (B2B) applications.

Using Organizations, you can:

- Represent teams, business customers, partner companies, or any logical grouping of users that should have different ways of accessing your applications, as organizations.

- Manage their membership in a variety of ways, including user invitation.

- Configure branded, federated login flows for each organization.

- Implement role-based access control, such that users can have different roles when authenticating in the context of different organizations.

- Build administration capabilities into your products, using Organizations APIs, so that those businesses can manage their own organizations.

Note that Organizations is currently only available to customers on our Enterprise and Startup subscription plans.

#### Log in to an organization

> Log into an organization by specifying the `organization` parameter when setting up the client:

Reviewer: ```suggestion
Log in to an organization by specifying the `organization` parameter when setting up the client:
```

```js
createAuth0Client({
  domain: '<AUTH0_DOMAIN>',
  client_id: '<AUTH0_CLIENT_ID>',
  redirect_uri: '<MY_CALLBACK_URL>',
  organization: '<MY_ORG_ID>'
});
```

You can also specify the organization when logging in:

```js
// Using a redirect
client.loginWithRedirect({
  organization: '<MY_ORG_ID>'
});

// Using a popup window
client.loginWithPopup({
  organization: '<MY_ORG_ID>'
});
```

### Accept user invitations

> Accept a user invitation through the SDK by creating a route within your application that can handle the invitation route, and log the user in by passing the `organization` and `invitation` parameters from this URL. You can either use `loginWithRedirect` or `loginWithPopup` as needed.

Reviewer: ```suggestion
Accept a user invitation through the SDK by creating a route within your application that can handle the invitation URL, and log the user in by passing the `organization` and `invitation` parameters from this URL to either `loginWithRedirect` or `loginWithPopup`.
```

Author: Thanks for the suggestion, I'm just conscious that perhaps it's now a little wordy with it all being one sentence 🤔

Reviewer: We can keep the last part as it was, my main concern was about the first part. It read weird to _create_ a route to _handle_ a route. Creating a route to handle a URL made more sense to me.

Author: Updated the text here 👍
